### PR TITLE
Homepage mobile design

### PR DESCRIPTION
**Pull request by sssuneeth**
- [X] Make **header** responsive
- [X] Add **bottom navbar** for mobile devices
- [ ] Make **bottom navbar** responsive

### CODE REVIEW

File: src/lib/store/responsive.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        case DESKTOP.matches:
+            return "desktop";
+        case WIDESCREEN.matches:
+            return "widescreen";
+        case FULLHD.matches:
+            return "fullhd";
+        default:
+            return null;
+    }
+};
```
Comment (baseplate-admin) [inline src/lib/store/responsive.ts:28]: Remove this

File: src/routes/+layout.svelte [hunk 2]
```diff
@@ -11,6 +11,7 @@
     import type { ModalSettings, ModalComponent } from "@skeletonlabs/skeleton";
 
     import { blur } from "svelte/transition";
+    import { responsive_mode } from "$store/responsive";
```
Comment (baseplate-admin) [inline src/routes/+layout.svelte:14]: Dont

**Review by baseplate-admin — CHANGES_REQUESTED** [review]

File: src/routes/+page.svelte [hunk 3]
```diff
@@ -413,7 +413,7 @@
             </div>
         </latest-animes>
 
-        <latest-episodes class="w-[21.5625vw]">
+        <latest-episodes class="w-[21.5625vw] hidden sm:block">
```
Comment (baseplate-admin) [inline src/routes/+page.svelte:416]: Dont use blocks

File: src/routes/+page.svelte [hunk 4]
```diff
@@ -468,7 +468,7 @@
         </latest-episodes>
 
         <navigation-card
-            class="relative mt-[3.4vw] h-[24.1325vw] w-[16.625vw] rounded-[0.875vw] bg-cover bg-center"
+            class="relative hidden sm:block mt-[3.4vw] h-[24.1325vw] w-[16.625vw] rounded-[0.875vw] bg-cover bg-center"
```
Comment (baseplate-admin) [inline src/routes/+page.svelte:471]: Dont use blocks

**Review by baseplate-admin — CHANGES_REQUESTED** [review]

File: src/routes/+page.svelte (hunk 4 above)
Comment (sssuneeth) [reply src/routes/+page.svelte:471]: Yea I'm gonna change that :)

**Comment by baseplate-admin** [comment]
image


Look at the bottom buttons. It doesn't look right

**Comment by baseplate-admin** [comment]
> Please send full pic? 

Its the full pic


> its half hidden?

Yes

**Comment by sssuneeth** [comment]
@baseplate-admin  **check it now**

**Pull request merged**